Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml

```diff
@@ -33,8 +33,8 @@
             <ToolBarTray x:Name="ToolsTray" Background="{StaticResource $BackgroundColor_LightGray}"  >
                 <ToolBar x:Name="toolbar" ToolBarTray.IsLocked="True" Band="1" Background="{StaticResource $BackgroundColor_LightGray}"
                      IsHitTestVisible="True" ScrollViewer.CanContentScroll="True" Width="{Binding ElementName=ToolsTray,Path=ActualWidth}">
-                    <Button x:Name="btnMarkAll" ToolTip="Mark All As InActive"  Visibility="Collapsed" Click="Btn_MarkUnMarkAll">
-                        <Image Source="{DynamicResource @UnCheckAllColumn_16x16.png}"/>
+                    <Button x:Name="btnMarkAll" ToolTip="Mark All As Active"  Visibility="Collapsed" Click="Btn_MarkUnMarkAll">
```

Comment:
Can not change it in ucGrid. Need to handle it from the specific place.
In this case the actions conversion wizard grid